From 5c96aeb0bb1379a6f2ed4ffe0fc03ef23a698d16 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Thu, 1 Sep 2016 17:16:43 +0300
Subject: [PATCH 0483/1345] pp3: Update PP3 driver version to NSS:16.08.0

commit  71a075a1b3d48e9d0c840500f514586f19791a86 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Changes from prevoius version: NSS:16.07.0
- change default number of TXQs to 1
- update emac-port shaper according to port-type
- fix output for "dev/stats" command
- remove default shaping and drop 2.5Gbps for EMAC0
- don't use SW queues 8-15 in frame 2 for data path
- change RU classes allocation
- mv_pp3_tx_done: print error with context
- use dev-name in netdev prints
- fix message unsupported autoneg-restart
- fw and drv version print stile adjust
- print fw error for no-callback request
- internal_debug action-on-error configuration
- bm pool refill on no-memory
- sysfs fw check initialization
- pp3: long multicast list handling
- stop upon bad cfh detection
- fw-version print without carriage-return
- gnss qstats - overflow fix
- txdone timer 1ms for TCP performance
- nss0 mtu change support

Change-Id: Ie994e86b4a367434a3afcb858f72b90a30e4bfcb
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32465
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
(cherry picked from commit 9c12411854db96625a1b307b0e6964e51ba68170)
Reviewed-on: http://vgitil04.il.marvell.com:8080/32495
Reviewed-by: Yan Markman <ymarkman@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../ethernet/marvell/pp3/platform/mv_pp3_version.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
index 5d1c619..f04e2d0 100644
--- a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
+++ b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
@@ -26,7 +26,7 @@
 static struct mv_pp3_version mv_pp3_driver_version = {
 	.name = "NSS",
 	.major_x = 16,
-	.minor_y = 7,
+	.minor_y = 8,
 	.local_z = 0,
 	.debug_d = 0
 };
-- 
1.7.9.5

